external-import/misp-feed/src/misp-feed.py

```diff
@@ -130,7 +130,7 @@ class Text:
 class MispFeed:
     def __init__(self):
         # Instantiate the connector helper from config
-        config_file_path = os.path.dirname(os.path.abspath(__file__)) + "/config.yml"
```

Comment:
Hey @ckane why did you remove this mecanism which is present in all connectors?

Reply:
The choice to remove this was so that the single copy of `misp-feed.py` could remain in the present folder. The current mechanism forces the script to use the `config.yml` that is co-located with it in the `misp-feed/` folder, which means I cannot run 3 instances of the same script with different `config.yml` files located in different folders and all 3 instance will only look at the `config.yml` located in the same directory as the Python script.

Making the above change allows me to `cd` into a different working directory, before calling each instance of the `misp-feed.py` script, and therefore have 3 running copies of the same module that pull down 3 feeds and run as 3 different connectors.

Reply:
I'll revert this change too, but I think it would be worth discussing whether all the connectors should behave this way, or if it should just look for the `config.yml` from the current working directory, and leave it up to the platform administrator to decide if they want the current working directory to be the same as the script's containing folder.